openfisca_aotearoa/variables/acts/social_security/schedule_4/part3.py

```diff
@@ -0,0 +1,324 @@
+
+    def formula_2018_11_26(people, period, parameters):
+        base = people("schedule_4__part3_1_e", period)
+        no_children = people("social_security__dependent_children", period) < 1
+        return base * no_children
+
+
+class schedule_4__part3_1_e_ii(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784861.html"
+    label = "Part 3 Supported Living Payment - Clause 1(e)(ii)"
+
+    def formula_2018_11_26(people, period, parameters):
+        base = people("schedule_4__part3_1_e", period)
+        has_children = people("social_security__dependent_children", period) > 0
+        return base * has_children
+
+
+# n.b.: this clause was repealed 2020-11-09 & folded into part3_1_e.
+class schedule_4__part3_1_f(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784861.html"
+    label = "Part 3 Supported Living Payment - Clause 1(f)"
+    end = "2020-11-09"
+
+    def formula_2018_11_26(people, period, parameters):
+        in_relationship = people("social_security__in_a_relationship", period)
+        partner_has_veterans_pension = people.family.any(
+            people.family.members("veterans_support__being_paid_a_veterans_pension", period.first_month),
+            role=entities.Family.PARTNER)
+        return in_relationship * partner_has_veterans_pension
+
+
+class schedule_4__part3_1_f_i(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784861.html"
+    label = "Part 3 Supported Living Payment - Clause 1(f)(i)"
+    end = "2020-11-09"
+
+    def formula_2018_11_26(people, period, parameters):
+        base = people("schedule_4__part3_1_f", period)
+        no_children = people("social_security__dependent_children", period) < 1
+        return base * no_children
+
+
+class schedule_4__part3_1_f_ii(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784861.html"
+    label = "Part 3 Supported Living Payment - Clause 1(f)(ii)"
+    end = "2020-11-09"
+
+    def formula_2018_11_26(people, period, parameters):
+        base = people("schedule_4__part3_1_f", period)
+        has_children = people("social_security__dependent_children", period) > 0
+        return base * has_children
+
+
+class schedule_4__part3_1_g(variables.Variable):
+    value_type = bool
+    entity = entities.Person
+    definition_period = periods.WEEK
+    reference = "https://www.legislation.govt.nz/act/public/2018/0032/latest/DLM6784861.html"
```

Comment:
As above - I'd suggest we also include a reference to the original version of the 2018 act so it's easier to track down the original formula and perhaps a link as well to the date it changed:
https://www.legislation.govt.nz/act/public/2018/0032/77.0/DLM6784861.html
https://www.legislation.govt.nz/act/public/2018/0032/140.0/DLM6784861.html